### ISSUE TITLE
BUG: 

### ISSUE THREAD

**Issue by dariagrudzien** [pacollins/hugo-future-imperfect-slim#219]
# Bug Report

## Describe the bug

Setting a favicon (at least non-SVG) doesn't seem possible.

## Expected behavior

When I put a `favicon.ico` file into `static/` folder and set `favicon = true` in `config.toml` I expect the favicon to appear on the browser tab.

## Screenshots

![Screen Shot 2021-01-31 at 15 10 26](https://user-images.githubusercontent.com/27728733/106386695-8591ad00-63d6-11eb-8659-779e5b3989d3.png)

## Environment

 - **Hugo Version:** : 0.79
 - **Browser and Version:** Chrome  88.0.4324.96
 - **Link to Project:**: https://github.com/cybermagnolia/cybermagnolia.com/blob/main/config.toml#L39

## Additional context

@pacollins Reopening #206 as I still can't figure it out. I combed GitHub to find a project which successfully set the favicon and found [this website](https://bytesician.netlify.app/) with [following config](https://github.com/robisetiapermadi/bytesician-hugo/blob/3f0257899a0919531a6b4cd92b222c7818469c98/config.toml#L36) I can see that the favicon is located in `static/` directly, named `favicon.ico`. 

So that's exactly what I set in my project, [remembering to increment the version as you suggested](https://github.com/cybermagnolia/cybermagnolia.com/blob/main/config.toml#L39)

At this point I think I tried every combination to troubleshoot this:

- I tried this favicon in a different project using the [Ananke theme](https://github.com/theNewDynamic/gohugo-theme-ananke/blob/master/exampleSite/config.toml#L30) and it worked right away
- I used a different favicon which works on the site using the `Ananke` theme and it doesn't work when I put it into this project.

I'm out of ideas at this point :)

**Comment by pacollins** [comment]
Can you turn SVG on for me? I think I see the problem.... If so, I'm dumb.

https://github.com/pacollins/hugo-future-imperfect-slim/blob/80efb2da6db0c9ffdf8609b216ba1b772b1a803b/layouts/partials/meta.html#L10-L14

**Comment by dariagrudzien** [comment]
@pacollins I did enable SVG with no success. Here's my current `params.meta` config:

```
    favicon             = true
    svg                 = true
    faviconVersion      = "8"
    msColor             = "#ffffff"
    iOSColor            = "#ffffff"
```

The favicon is called `favicon.ico` and placed in `static` directory directly. I used https://favicon.io/favicon-converter/ to convert a png file into ico in case I was making a mistake with format conversion. Nothing changes.

**Comment by pacollins** [comment]
It does work, you are just not putting the favicon in the right directory. It should be in `static/favicon` thus be `static/favicon/favicon.ico`. Upon inspecting the wiki, it looks like my wording is not accurate to what I intended. There is not support for a root directory favicon. That would need to be a PR adding `{{ else }}` at line 14.

https://github.com/pacollins/hugo-future-imperfect-slim/blob/80efb2da6db0c9ffdf8609b216ba1b772b1a803b/layouts/partials/meta.html#L12

**Comment by dariagrudzien** [comment]
You're right. I placed the icon in `static/favicon/` and set this `params.meta` config:
